pos_partner_firstname/static/src/js/models.js

```diff
@@ -0,0 +1,7 @@
+odoo.define("pos_partner_firstname.models", function (require) {
+    "use strict";
+
+    var models = require("point_of_sale.models");
+
+    models.load_fields("res.partner", ["is_company", "firstname", "lastname"]);
+});
```

Comment:
```suggestion

    return models;
});
```

To play nice with other modules